scripts/run-tests.sh

```diff
@@ -136,18 +136,18 @@ get_pytest_files
 test_result=0
 
 convert_and_test "qemux86_64" \
-                 "release-1" \
-                 "${UBUNTU_IMAGE_URL}" \
-                 "${UBUNTU_IMAGE}" \
-                 "${UBUNTU_IMAGE}.gz" \
-                 "configs/qemux86-64_config" || test_result=$?
+  "release-1" \
```

Comment:
I don't really know if these should follow 2-space indent or not?

Reply:
Might as well